modules/apps/27-interchain-accounts/host/types/params.go

```diff
@@ -34,7 +34,7 @@ func NewParams(enableHost bool, allowMsgs []string) Params {
 
 // DefaultParams is the default parameter configuration for the host submodule
 func DefaultParams() Params {
-	return NewParams(DefaultHostEnabled, nil)
+	return NewParams(DefaultHostEnabled, []string{"*"})
```

Comment:
LGTM, I think it'd be nice to add a const for this string since we use it in two places now

Reply:
added a commit to store it as const